src/Table.tsx

```diff
@@ -609,6 +610,9 @@ function Table<RecordType extends DefaultRecordType>(props: TableProps<RecordTyp
     <ColGroup colWidths={flattenColumns.map(({ width }) => width)} columns={flattenColumns} />
   );
 
+  const captionElement =
+    caption != null ? <caption className={`${prefixCls}-caption`}>{caption}</caption> : undefined;
```

Comment:
```suggestion
    caption !== null && caption !== undefined ? <caption className={`${prefixCls}-caption`}>{caption}</caption> : undefined;
```

Reply:
`!= null` will find it caption is `null` and `undefined`. https://jsfiddle.net/mellis481/8fvtyzL7/

Reply:
@afc163 Let's get this PR merged.  Are you OK if I use `!!` for the reason above?

Reply:
`!=` should not be used in any circumstance

Reply:
> `!=` should not be used in any circumstance

That's an interesting assertion.  Could you share information or an article explaining why?

In any event, I've made the change you requested and I believe this was the only outstanding comment.